File: app/src/main/java/com/example/okta/moviedb_iak/activity/SplashscreenActivity.java
package com.example.okta.moviedb_iak.activity;

import android.content.Intent;
import android.os.Handler;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

import com.example.okta.moviedb_iak.R;

public class SplashscreenActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_splashscreen);

        new Handler().postDelayed(new Runnable() {
            @Override
            public void run() {
                Intent mainInten = new Intent(getApplicationContext(),MainActivity.class);
                startActivity(mainInten);
            }
        },2000);
    }
}

File: app/src/main/java/com/example/okta/moviedb_iak/activity/SplashscreenActivity.kt
package com.example.okta.moviedb_iak.activity

import android.content.Intent
import android.os.Handler
import android.support.v7.app.AppCompatActivity
import android.os.Bundle

import com.example.okta.moviedb_iak.R

class SplashscreenActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_splashscreen)

        Handler().postDelayed({
            val mainInten = Intent(applicationContext, MainActivity::class.java)
            startActivity(mainInten)
            finish()
        }, 2000)
    }
}
